source/nodejs/adaptivecards-controls/src/constants.ts

```diff
@@ -1,9 +1,42 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+/**
+ * @deprecated Use Constants.keys.tab instead.
+ */
 export const KEY_TAB = "Tab";
+/**
+ * @deprecated Use Constants.keys.enter instead.
+ */
 export const KEY_ENTER = "Enter";
+/**
+ * @deprecated Use Constants.keys.escape instead.
+ */
 export const KEY_ESCAPE = "Escape";
+/**
+ * @deprecated Use Constants.keys.space instead.
+ */
 export const KEY_SPACE = " ";
+/**
+ * @deprecated Use Constants.keys.up instead.
+ */
 export const KEY_UP = "Up";
+/**
+ * @deprecated Use Constants.keys.down instead.
+ */
 export const KEY_DOWN = "Down";
+/**
+ * @deprecated Use Constants.keys.delete instead.
+ */
 export const KEY_DELETE = "Delete";
```

Comment:
The deprecated constants need to be static members of the Constants class, otherwise we're breaking code that refers to them as `Constants.KEY_UP` for instance.

Reply:
These are (and previously were) exported as top-level constants in the '/constants.ts' module, which were then exported as top-level constants in the 'adaptivecards-controls' package. So, an external library had to have used them like this:

```
import * as Controls from "adaptivecards-controls"

Controls.KEY_ENTER
```

I did not modify that. The new Constants class in the '/constants.ts' module has to be used like this:

```
import * as Controls from "adaptivecards-controls"

Controls.Constants.key.enter
```

If I move the deprecated constants into that Constants class, the old usage would be broken.

Reply:
Ignore this - my mistake, what you have works.